packages/extension/src/newtab/ShortcutLinks/experiments/ShortcutLinksUIV1.tsx

```diff
@@ -0,0 +1,212 @@
+import React, { FC, PropsWithChildren } from 'react';
+import {
+  ClearIcon,
+  MenuIcon,
+  PlusIcon,
+} from '@dailydotdev/shared/src/components/icons';
+import {
+  Button,
+  ButtonSize,
+  ButtonVariant,
+} from '@dailydotdev/shared/src/components/buttons/Button';
+
+import { useActions } from '@dailydotdev/shared/src/hooks';
+import { IconSize } from '@dailydotdev/shared/src/components/Icon';
+import classNames from 'classnames';
+import { combinedClicks } from '@dailydotdev/shared/src/lib/click';
+import { ActionType } from '@dailydotdev/shared/src/graphql/actions';
+import { cloudinary } from '@dailydotdev/shared/src/lib/image';
+import { useThemedAsset } from '@dailydotdev/shared/src/hooks/utils';
+
+const pixelRatio = globalThis?.window.devicePixelRatio ?? 1;
+const iconSize = Math.round(24 * pixelRatio);
+
+type ShortcutLinksV1Props = {
+  onLinkClick: () => void;
+  onMenuClick: React.MouseEventHandler<Element>;
+  onOptionsOpen: () => void;
+  onV1Hide: () => void;
+  shortcutLinks: string[];
+  shouldUseListFeedLayout: boolean;
+  showTopSites: boolean;
+  toggleShowTopSites: () => void;
+};
+
+const ShortCutV1Placeholder: FC<{
```

Comment:
Not pro/against but we never really use FC format anywhere.

Reply:
Okay! If preferred I can switch to `function Name(props: Props): ReactElement` is that better?

Reply:
It's more inline, but honestly don't mind.
Maybe we can open it up on [#guild-web](https://dailydotdev.slack.com/archives/C02E2C3C13R) slack channel.

I do prefer 1 or the other just so everyone uses the same.

Reply:
Okay, just keeping the old one for now and then will see ;)